storage/s3.go

```diff
@@ -693,3 +713,36 @@ func (a *writeAtAdapter) Write(p []byte) (int, error) {
 	a.offset += int64(n)
 	return n, nil
 }
+
+type Metadata map[string]string
+
+func (m Metadata) SetACL(acl string) {
+	m["ACL"] = acl
+}
+func (m Metadata) ACL() string {
+	return m["ACL"]
+}
+func (m Metadata) SetStorageClass(class string) {
+	m["StorageClass"] = class
+}
+func (m Metadata) StorageClass() string {
+	return m["StorageClass"]
```

Comment:
I think it would be safer to check if the keys exists
```go
value, ok := m["key"]
if !ok {
	return ""
}

return value
```
There are already checks for empty string in
https://github.com/peak/s5cmd/blob/608ee4dd730041b8ba76581e65408b884ccdf3bb/storage/s3.go#L310-L311
https://github.com/peak/s5cmd/blob/608ee4dd730041b8ba76581e65408b884ccdf3bb/storage/s3.go#L315-L316